src/main/java/transforms/GeographyPointTransform.java

```diff
@@ -0,0 +1,62 @@
+package transforms;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.openlattice.shuttle.transformations.Transformation;
+import com.openlattice.shuttle.util.Constants;
+import com.openlattice.shuttle.util.Parsers;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.List;
+import java.util.Map;
+
+public class GeographyPointTransform extends Transformation<Map<String, String>> {
+    private final List<Transformation> latTransforms;
+    private final List<Transformation> lonTransforms;
+
+    /**
+     * Represents a transformation to concatenate values *resulting from other transformations*.
+     * The difference with ConcatTransform is that the input is transformations, and the resulting string will be
+     * concatenated using the requested separator. Empty cells are skipped.  If all
+     * are empty, null is returned.
+     *
+     * @param latTransforms: list of transformations to get latitude
+     * @param lonTransforms: list of transformations to get longitude
+     */
+    @JsonCreator
+    public GeographyPointTransform(
+            @JsonProperty(Constants.LAT_TRANSFORMS) List<Transformation> latTransforms,
+            @JsonProperty(Constants.LON_TRANSFORMS) List<Transformation> lonTransforms) {
+        this.latTransforms = latTransforms;
+        this.lonTransforms = lonTransforms;
+    }
+
+    @Override
+    public Object apply(Map<String, String> row) {
+
+        // LATITUDE TRANSFORMATIONS
+        Object lat = row;
+        for (Transformation t : latTransforms) {
+            lat = t.apply(lat);
```

Comment:
what's the purpose of overwriting the lat value for every transformation? Is it the type of situation where there will be multiple potential ways to get the latitude value, and probably only one of them will work?

Reply:
This is for when the input is not clean and we need to add more transformations to clean it.  (Eg. when missing value == "MISSING")